web/pages/database/[databaseName]/compare/[refName]/[diffRange].tsx

```diff
@@ -5,14 +5,14 @@ import { GetServerSideProps, NextPage } from "next";
 
 type Params = RefParams & {
   diffRange: string;
+  tableName?: string;
```

Comment:
I wouldn't expect there to be any params/props changes in this PR? Where are all of these coming from?